dashboard/src/components/resource-list/ResourceListView.js

```diff
@@ -494,7 +494,7 @@ const NoDataMessage = ({ type }) => {
   const classes = useStyles();
 
   function redirect() {
-    window.location.href = 'https://docs.featureform.com/quickstart';
+    window.location.href = 'https://docs.featureform.com/quickstart-local';
```

Comment:
Might wanna change this to just be the overview
https://docs.featureform.com/getting-started/overview